.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on:
+  pull_request:
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Build WASM release
+      run: make docker_release
+
+    # This step will evaluate the repo status and exit if found changes
```

Comment:
this should detect the changes in the binary, right (as the binary is pushed)? 
Maybe worth to mention that the status diff is for this purpose?